packages/babel-preset-gatsby-package/src/index.js

```diff
@@ -54,6 +74,13 @@ function preset(context, options = {}) {
       r(`@babel/plugin-transform-runtime`),
       r(`@babel/plugin-syntax-dynamic-import`),
       IS_TEST && r(`babel-plugin-dynamic-import-node`),
+      availableCompilerFlags.length && [
+        r(`./babel-transform-compiler-flags`),
+        {
+          flags: parsedCompilerOptions,
+          availableFlags: availableCompilerFlags,
+        },
+      ],
```

Comment:
Custom plugin is fine but wouldn't https://github.com/FormidableLabs/babel-plugin-transform-define achieve same thing without us having to maintain custom babel plugin/trasnform?

Reply:
I wanted a custom one because this does the minimal work and is all that we need. transform-define does more and won't have the same heuristics.